guide/designing-products/user-research-resources.md

```diff
@@ -37,29 +37,53 @@ Below is a collection of published user research findings around Bitcoin, from a
 ## Research papers
 
 ###### [Non-adoption of crypto-assets: Exploring the role of trust, self-efficacy, and risk](https://voskart.de/pdf/crypto_adoption.pdf)
+
+*A survey of 404 users and non-users of Bitcoin about the roles of trust, self-efficacy, and risk on adoption intention.*
+
 Published in May 2021 by Artemij Voskobojnikov, Svetlana Abramova, Konstantin Beznosov, and Rainer Böhme.
 
 ###### [Bits Under the Mattress: Understanding Different Risk Perceptions and Security Behaviors of Crypto-Asset Users](https://voskart.de/pdf/bits_under_mattress.pdf)
+
+*A survey of security behaviors among three Bitcoin user archetypes, showing that cypherpunks tend to use self-managed security solutions, whereas hodlers and rookies use either custodial solutions or non-custodial wallets.*
+
 Published in February 2021 by Svetlana Abramova, Artemij Voskobojnikov, Konstantin Beznosov, and Rainer Böhme.
 
 ###### [The U in Crypto Stands for Usable: An Empirical Study of User Experience with Mobile Cryptocurrency Wallets](https://voskart.de/pdf/u_in_crypto.pdf)
+
+*An analysis of app reviews of the top five mobile cryptocurrency wallets suggesting that both new and experienced users struggle with general and domain-specific UX issues that might lead to dangerous errors and irreversible monetary losses.*
+
 Published in January 2021 by Artemij Voskobojnikov, Oliver Wiese, Masoud Mehrabi Koushki, Volker Roth, and Konstantin Beznosov.
 
 ###### [User Perception of Bitcoin Usability and Security across Novice Users](https://www.researchgate.net/publication/331286661_User_Perception_of_Bitcoin_Usability_and_Security_across_Novice_Users)
+
+*A survey of undergraduate students at Keele University concluding that Bitcoin was still in its infancy at the time of writing and required user education and a new way of thinking. The study recommends user-centered design to develop users’ mental models and deepen their understanding of digital payment technology.*
```

Comment:
```suggestion
This paper conclude that Bitcoin, as a crypto-currency, is still in its infancy and requires user education and a new way of thinking. The study recommends developing users’ mental models to deepen developers’ understanding of anonymous digital payment technology and improve user-centered design. The paper also makes recommendations with respect to e-government services that may be developed relying on crypto-currencies.
```

Reply:
We had this part in the original summary but removed. To me, it's just really poorly written.
- I find it misleading not to mention that this is based on a very small sample of students from one university
- Bitcoin is not anonymous
- What does "Bitcoin, as a crypto-currency" mean? What else can Bitcoin be?
- How can the development of the mental models of users deepen the understanding developers have of digital payment technology? When users understand Bitcoin better, then developers understand digital payment tech better?
- The text in the paper about e-government is fluff, there's nothing specific there

TBH, I am a bit frustrated with how some of these paper summarize their content. It's a lot of "difficult language fluff" with little actual meaning.

Reply:
If the results of the papers are not accurate / misleading in ways like you mentioned - should we be including them in the guide at all?

Reply:
Good question, maybe not this one.

After looking through all of them, I've found the actual survey data usually very interesting. The conclusions are more what I've had problems with. This could be my bias though. @johnsBeharry said he'd also do a deeper review of each paper, maybe he'll draw different conclusions...

Reply:
I noticed with the more older papers as well a lot of the summaries may be quite dated. The industry moves so fast it's hard to draw the same conclusions today.

Reply:
For sure. That's why I ordered the research by date and made sure the dates are easily visible. Unfortunately, the page would be pretty empty if we only showed recent research.